ani-cli

```diff
@@ -337,6 +337,9 @@ open_episode () {
 					printf "${c_red}\nCannot reach servers!"
 					rm "${logfile}.new"
 				else
+                                        if pgrep mpv; then
+                                            pkill mpv
+                                        fi
 					nohup $player_fn "$play_link" > /dev/null 2>&1 &
```

Comment:
Instead of using `pgrep` you can get the PID of last ran program with
```
PID=$!
```
So, just before running `mpv` we can kill the previous instance like this
```
kill $PID
```

It will look something like this
```
kill $PID
nohup stuff
PID=$!
```

Reply:
I figured it out on how to implement that, though having `kill $PID` without having any defined process will return an error. Instead I used this:
```
if ps "$PID" &> /dev/null; then
    kill "$PID"
fi
```
so that it checks if a previous instance is created.

Reply:
Actually, we don't need to check for a previous instance. We can make `kill` silent
```
kill "$PID" >/dev/null 2>&1
```